### ISSUE TITLE
Upgrading from 0.0.69 to 0.0.71 leads to `yarn check` errors

### ISSUE THREAD

**Issue by kaiyoma** [williaster/data-ui#151]
I've been using version 0.0.69 of `histogram` and `xy-chart` for a little while now. When I tried to upgrade to version 0.0.71 this morning, I started getting `yarn check` errors immediately:

```
$ yarn check
yarn check v1.12.3
info fsevents@1.2.4: The platform "win32" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
error "@data-ui/histogram##@vx/curve@^0.0.165" doesn't satisfy found match of "@data-ui\\histogram#@vx\\curve@0.0.140"
error "@data-ui/histogram##@vx/shape@^0.0.179" doesn't satisfy found match of "@data-ui\\histogram#@vx\\shape@0.0.140"
warning "@data-ui/xy-chart##@vx/axis@^0.0.168" could be deduped from "0.0.168" to "@vx\\axis@0.0.168"
warning "@data-ui/xy-chart##@vx/glyph@^0.0.165" could be deduped from "0.0.165" to "@vx\\glyph@0.0.165"
warning "@data-ui/xy-chart##@vx/gradient@^0.0.165" could be deduped from "0.0.165" to "@vx\\gradient@0.0.165"
warning "@data-ui/xy-chart##@vx/pattern@^0.0.165" could be deduped from "0.0.165" to "@vx\\pattern@0.0.165"
warning "@data-ui/xy-chart##@vx/responsive@^0.0.165" could be deduped from "0.0.165" to "@vx\\responsive@0.0.165"
warning "recompose#@babel/runtime@^7.0.0" could be deduped from "7.1.5" to "@babel\\runtime@7.1.5"
error "@data-ui/histogram#@vx/event##@vx/point@0.0.165" doesn't satisfy found match of "@data-ui\\histogram#@vx\\point@0
.0.136"
warning "webpack-dev-server#sockjs-client#url-parse@^1.1.8" could be deduped from "1.4.4" to "url-parse@1.4.4"
error "@data-ui/xy-chart#@vx/axis##@vx/group@0.0.165" doesn't satisfy found match of "@vx\\axis#@vx\\group@0.0.140"
error "@data-ui/xy-chart#@vx/axis##@vx/point@0.0.165" doesn't satisfy found match of "@vx\\axis#@vx\\point@0.0.136"
error "@data-ui/histogram#@vx/axis##@vx/point@0.0.165" doesn't satisfy found match of "@data-ui\\histogram#@vx\\point@0.
0.136"
error "@data-ui/histogram#@vx/axis##@vx/shape@0.0.179" doesn't satisfy found match of "@data-ui\\histogram#@vx\\shape@0.
0.140"
error "@data-ui/histogram#@vx/tooltip##@vx/bounds@0.0.165" doesn't satisfy found match of "@data-ui\\histogram#@vx\\boun
ds@0.0.140"
warning "react-redux#hoist-non-react-statics#react-is@^16.3.2" could be deduped from "16.6.3" to "react-redux#react-is@1
6.6.3"
error "@vx/axis#@vx/shape##@vx/curve@0.0.165" doesn't satisfy found match of "@vx\\axis#@vx\\curve@0.0.140"
error "@vx/axis#@vx/shape##@vx/group@0.0.165" doesn't satisfy found match of "@vx\\axis#@vx\\group@0.0.140"
error "@vx/axis#@vx/shape##@vx/point@0.0.165" doesn't satisfy found match of "@vx\\axis#@vx\\point@0.0.136"
warning "@storybook/react#webpack#uglifyjs-webpack-plugin##webpack@^1.9 || ^2 || ^2.1.0-beta || ^2.2.0-rc || ^3.0.0" cou
ld be deduped from "3.12.0" to "@storybook\\react#webpack@3.12.0"
warning "eventsource#original#url-parse@^1.4.3" could be deduped from "1.4.4" to "url-parse@1.4.4"
info Found 10 warnings.
error Found 11 errors.
```

**Comment by williaster** [comment]
Hey @kaiyoma thanks for checking out `@data-ui`!

Libraries shouldn't have lock files so we've not committed them, which makes me think that a mismatch between installed packages and a `yarn.lock` might not be a problem with `@data-ui`. Have you deleted your `yarn.lock` and tried a fresh install? If that doesn't work then it's possible the lock file was included in the published package 🤔

**Comment by williaster** [comment]
thanks for the update 👍 

Even though we've not committed it, I'd guess that the `yarn.lock` is being included in the npm package because it is not explicitly included in a `.gitignore` / `.npmignore` file. I'll try to release a new version in the next day or two with a fix for this.

**Comment by williaster** [comment]
@kaiyoma this should be resolved in `0.0.74` lmk if you still are experiencing issues 👍

**Comment by kaiyoma** [comment]
Yup, we upgraded to the newest version and after some clean installs (e.g. `rm -rf node_modules && yarn install`) everything is working again. Thanks!
